### PR TITLE
bitwarden: 2023.9.0 -> 2023.9.3

### PR DESCRIPTION
**Pull request by dotlambda**
## Description of changes
Diff: https://github.com/bitwarden/clients/compare/desktop-v2023.9.0...desktop-v2023.9.3

Changelog: https://github.com/bitwarden/clients/releases/tag/desktop-v2023.9.3

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

closes https://github.com/NixOS/nixpkgs/pull/260699

### CODE REVIEW

File: pkgs/tools/security/bitwarden/default.nix [hunk 1]
```diff
@@ -141,18 +144,19 @@ in buildNpmPackage' rec {
...
       dir=$out/share/icons/hicolor/"''${icon%.png}"/apps
       mkdir -p "$dir"
       cp "$icon" "$dir"/${icon}.png
     done
     popd
+
+    runHook postInstall
   '';
 
+  desktopItems = [ desktopItem ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/bitwarden/default.nix:158]: we could inline desktopItem but that's a nit, feel free to merge it without this

**Review by SuperSandro2000 — APPROVED** [review]
already tested just the update commit and that worked thanks

**Review by amarshall — APPROVED** [review]
Looks good, tested build and that desktop entry is present in output still.

**Comment by adingbatponder** [comment]
Trying to install bitwarden 
Get this:
building the system configuration...
error: Package ‘electron-24.8.6’ in /nix/var/nix/profiles/per-user/root/channels/nixos/pkgs/development/tools/electron/binary/generic.nix:27 is marked as insecure, refusing to evaluate.


       Known issues:
        - Electron version 24.8.6 is EOL

       You can install it anyway by allowing this package, using the
       following methods:

       a) To temporarily allow all insecure packages, you can use an environment
          variable for a single invocation of the nix tools:

            $ export NIXPKGS_ALLOW_INSECURE=1

        Note: For `nix shell`, `nix build`, `nix develop` or any other Nix 2.4+
        (Flake) command, `--impure` must be passed in order to read this
        environment variable.

       b) for `nixos-rebuild` you can add ‘electron-24.8.6’ to
          `nixpkgs.config.permittedInsecurePackages` in the configuration.nix,
          like so:

            {
              nixpkgs.config.permittedInsecurePackages = [
                "electron-24.8.6"
              ];
            }

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
          ‘electron-24.8.6’ to `permittedInsecurePackages` in
          ~/.config/nixpkgs/config.nix, like so:

            {
              permittedInsecurePackages = [
                "electron-24.8.6"
              ];
            }
(use '--show-trace' to show detailed location information)

**Comment by luftmensch-luftmensch** [comment]
Hi there, is possible to fix the the backport on 23.05, so there is no need to enable insicure package?

**Pull request merged**